.ci/fullDownstreamBuild.properties

```diff
@@ -1,8 +1,8 @@
-goals.default.upstream=-e -T1C clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true
+goals.default.upstream=-e --builder smart -T1.0C clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true
 goals.optaplanner.upstream=-e clean install -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dspotbugs.skip=true -Drevapi.skip=true
-goals.default.current=-e -nsu -T1C clean install -Dfull -DskipTests
+goals.default.current=-e -nsu --builder smart -T1.0C clean install -Dfull -DskipTests
 goals.optaplanner.current=-e -nsu clean install -Dfull -DskipTests
-goals.droolsjbpm-integration.current=-e -nsu -B -T1C clean install -Dfull -Pjenkins-pr-builder -DskipTests
-goals.kie-wb-common.current=-e -nsu -T1C clean install -Dfull -DskipTests
+goals.droolsjbpm-integration.current=-e -nsu -B --builder smart -T1.0C clean install -Dfull -Pjenkins-pr-builder -DskipTests
+goals.kie-wb-common.current=-e -nsu --builder smart -T1.0C clean install -Dfull -DskipTests
```

Comment:
shouldn'ŧ be shorter to use just `-T1C`

Reply:
--builder smart -T1.0C  is needed because of this takari extensions - I thnk there i no difference between -T1C and -T1.0C

Reply:
just -T1C is shorter, no big deal, but pointing that out ;) They are the same I agree